src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1550,6 +1550,12 @@ public Article updateArticle(Article article) {
                 JSON, json(Collections.singletonMap("article", article))), handle(Article.class, "article")));
     }
 
+    public Translation createArticleTranslation(Long articleId, Translation translation) {
+        checkHasArticleId(articleId);
```

Comment:
because in create article the translation has no id (only post id)
but. we do need to make sure article id is not null